### PR TITLE
affected entities.

### PR DESCRIPTION
**Pull request by PawelPawelec-RDX**

### CODE REVIEW

File: src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/WriteHelper.cs [hunk 1]
```diff
@@ -194,6 +194,7 @@ public async Task<int> CopyLedgerTransaction(ICollection<LedgerTransaction> enti
             await writer.WriteAsync(lt.IsEndOfEpoch, NpgsqlDbType.Boolean, token);
             await writer.WriteNullableAsync(lt.FeePaid?.GetSubUnitsSafeForPostgres(), NpgsqlDbType.Numeric, token);
             await writer.WriteNullableAsync(lt.TipPaid?.GetSubUnitsSafeForPostgres(), NpgsqlDbType.Numeric, token);
+            await writer.WriteNullableAsync(lt.AffectedGlobalEntities, NpgsqlDbType.Array | NpgsqlDbType.Bigint, token);
```
Comment (KrzysztofLabus-RDX) [inline src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/WriteHelper.cs:197]: Open question: do we want nullable array or maybe empty array? Sounds like latter suits us better?

File: src/RadixDlt.NetworkGateway.PostgresIntegration/Services/TransactionQuerier.cs [hunk 2]
```diff
@@ -186,6 +187,25 @@ public async Task<TransactionPageWithoutTotal> GetTransactionStream(TransactionS
             }
         }
 
+        if (request.SearchCriteria.AffectedGlobalEntities.Any())
+        {
+            userKindFilterImplicitlyApplied = true;
```
Comment (KrzysztofLabus-RDX) [inline src/RadixDlt.NetworkGateway.PostgresIntegration/Services/TransactionQuerier.cs:192]: Are we sure about that? I think non-user TXs can affect various entities.

File: src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/WriteHelper.cs (hunk 1 above)
Comment (PawelPawelec-RDX) [reply src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/WriteHelper.cs]: fixed.

File: src/RadixDlt.NetworkGateway.PostgresIntegration/Services/TransactionQuerier.cs (hunk 2 above)
Comment (PawelPawelec-RDX) [reply src/RadixDlt.NetworkGateway.PostgresIntegration/Services/TransactionQuerier.cs]: fixed, you're right.

**Review by KrzysztofLabus-RDX — APPROVED** [review]

**Pull request merged**